### PR TITLE
Bug 1244020 - [TV][2.5] Reset preview window open times

### PR DESCRIPTION
**Pull request by rickychien**
https://bugzilla.mozilla.org/show_bug.cgi?id=1244020

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c996cd727256e4fe232d69fc30eac93da6c0d4b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=526d9d5cb85130a96d26bf926e7737ffa7bfd908

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3fe2bd1b9e6fe751f70b81dd8d32acb636b2015b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ab2d6ca37fdbd9d219a9ad36989b9c326f926795

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9a3840a2369344dd7c62e4e529904e4a32a99326

File: tv_apps/smart-system/js/app_install_manager.js [hunk 1]
```diff
@@ -349,6 +350,26 @@ var AppInstallManager = {
...
+    if (openedTimes[id]) {
+      openedTimes[id] = 0;
+      localStorage.setItem(this.PREVIEW_OPENED_TIMES_KEY,
+        JSON.stringify(openedTimes));
+    }
+  },
+
+  increasePreviewOpenedTimes: function(id) {
+    var openedTimes = JSON.parse(localStorage.getItem(
+        AppInstallManager.PREVIEW_OPENED_TIMES_KEY) || '{}');
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_install_manager.js]: `this.PREVIEW_OPENED_TIMES_KEY`

File: tv_apps/smart-system/js/preview_window.js [hunk 2]
```diff
@@ -9,8 +9,6 @@
 /* global SystemBanner */
 
 (function(exports) {
-  const PREVIEW_OPENED_TIMES_KEY = 'preview-opened-times';
-  const PREVIEW_OPENED_TIMES_TO_HINT = 3;
```
Comment (luke-chang) [inline tv_apps/smart-system/js/preview_window.js]: You forgot to put this constant variable back in app_install_manager.

File: tv_apps/smart-system/js/app_install_manager.js [hunk 3]
```diff
@@ -349,6 +350,26 @@ var AppInstallManager = {
...
+    openedTimes[id] = openedTimes[id] || 0;
+    openedTimes[id]++;
+    localStorage.setItem(AppInstallManager.PREVIEW_OPENED_TIMES_KEY,
```
Comment (luke-chang) [inline tv_apps/smart-system/js/app_install_manager.js]: same as above.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4b132ce34f40499145238c683b1541d8f35d3445

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=76d880a103836fc4f79a9e331ec2cd24d2d0b75b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ec501b45cac9d1bdc50848c4a4e4cc399cc02ef1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=681a127c50fdeb1b7aef0e0a0b66b19c0763eaa3

**Pull request merged**